fab/fab/services/templates/supervisor_handle_survey_actions.conf

```diff
@@ -1,12 +0,0 @@
-[program:{{ project }}-{{ environment }}-handle_survey_actions]
-directory={{ code_current }}/
-command={{ virtualenv_current }}/bin/python manage.py handle_survey_actions
-user={{ sudo_user }}
-numprocs=1
-autostart=true
-autorestart=true
-stdout_logfile={{ log_dir }}/handle_survey_actions.log
-redirect_stderr=true
-stderr_logfile={{ log_dir }}/handle_survey_actions.error.log
-startsecs=0
-stopwaitsecs=10
```

Comment:
also just unused?

Reply:
This wasn't unused, but a lot of these were using the exact same template with minor variations. I combined them into a single template https://github.com/dimagi/commcare-cloud/pull/1424/commits/479f0b045c7753fdd4ea59015624b6052ffb18a7#diff-0bce1acd60cfc963a489f95f234323eeR18